dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/api/Connection.java

```diff
@@ -103,14 +106,19 @@ private Bootstrap create() {
             @Override
             protected void initChannel(SocketChannel ch) {
                 ch.attr(CONNECTION).set(Connection.this);
-                // TODO support SSL
+
+                SslContext sslContext = SslContexts.buildClientSslContext(url);
+
+                if (getUrl().getParameter(SSL_ENABLED_KEY, false)) {
+                    ch.pipeline().addLast("negotiation", SslHandlerInitializer.sslClientHandler(sslContext, connectionHandler));
```

Comment:
It is enough to delegate SSL handling to PU handler rather than add it to pipepline firstly

Reply:
Done.
Migrate to SslServerTlsHandler.